06 piškvorky/piskvorky1D.py

```diff
@@ -0,0 +1,77 @@
+from random import randrange
+import time
+
+# TICK-TACK-TOE
+
+
+def score(area):  # fce get list(area) and return result in accordance with area
+
+    # draw
+    if '-' not in area:
+        print(' "!" ')
+        quit()
```

Comment:
Tieto sú tiež zbytočné.

Reply:
Pokud to neukončím, tak ten kód pokračuje. Sice to napíše, že vyhrál "x" , ale zeptá se i na další políčko

Reply:
To je spôsobené tým, že funkciu `score` voláš vo while cykle ktorý nikdy neskončí, takže ten `quit` ukončuje celý program. To nie je veľmi systematické riešenie. Vo funkcii `tick_tack_toe_1D` by mal byť ten while bežať do vtedy kým sa hra dá ešte hrať (t.j. nikto nevyhral a v poli sú ešte nejaké prázdne políčka). Pokiaľ niekto vyhrá alebo je vyčerpané celé pole, while by sa mal preprušiť pomocou break a až po jeho ukončení by som vyprintovala výsledok.